core/src/main/java/de/danoeh/antennapod/core/preferences/UserPreferences.java

```diff
@@ -54,6 +54,7 @@ private UserPreferences(){}
     private static final String PREF_SHOW_DOWNLOAD_REPORT = "prefShowDownloadReport";
     public static final String PREF_BACK_BUTTON_BEHAVIOR = "prefBackButtonBehavior";
     private static final String PREF_BACK_BUTTON_GO_TO_PAGE = "prefBackButtonGoToPage";
+    public static final String PREF_QUEUE_SORT_ORDER = "prefQueueSortOrder";
```

Comment:
In my opinion, it would be better to have two separate settings. One that stores if sorting is enabled and one that stores the order. That allows to toggle automatic sort without having to re-select the order.

Reply:
I agree, I will change this.